src/int.rs

```diff
@@ -109,14 +109,14 @@ macro_rules! IVec2 {
 
             #[inline]
             pub fn mag_sq(&self) -> $t {
-                self.x.mul_add(self.x, self.y * self.y)
+                (self.x * self.x) + (self.y * self.y)
             }
 
             #[inline]
             pub fn mul_add(&self, mul: $n, add: $n) -> Self {
```

Comment:
For the explicit `mul_add` function, I'll want to keep the actual `mul_add` here in case the user wants to use it.